### DISCUSSION TITLE
Message Manager Question

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#893] [Q&A] [answered]
I am trying to determine if there is a communication problem between Nixie & an Intellichlor cell in my system.  I am able to control an Intelliflow VSF pump but not configure the salt cell.  The salt cell LEDs are green, indicating good water flow, good salt level, & 20 % salt generation.

In the dashPanel Comm tab I can see packet counts increasing over time for both sent & received packets.  The error rate is about 1%.

I started message manager to see if something is amiss.  When I hit the hamburger at the top right of the Messages window to start the live feed, no messages are displayed  and the hamburger turns orange.  Hitting the burger again inverts the icon color but does not cause any messages to appear.

Am I screwed somehow?  What packets are showing up on the Comms tab that can't be displayed in Message Manager?  And finally, how can I control the pump if I can't see any messages?

I'm feeling a bit dazed & confused at this point & am wondering, dude, where's my comms?

Thanks for any thoughts on how to proceed!

**Comment by truvec** [comment, marked as answer]
Probably don't have packets turned on.

<img width="457" alt="Screenshot 2024-01-31 at 2 00 17 AM" src="https://github.com/tagyoureit/nodejs-poolController/assets/239790/3b189368-0eb5-4d70-a537-7ce063638125">

https://github.com/rstrouse/nodejs-poolController-dashPanel/wiki/Message-Manager

**Reply by FlaMike** [reply]
Thanks!  I am getting bidirectional packets between the pump & the "plugin."  I assume that's njsPC.

**Reply by FlaMike** [reply]
I think this section is what messed me up.  There's a typo in it.  The hamburger is on the upper _left_ of the display window, not the upper right.  I was trying to access it via the  start/stop button which is almost hamburger-like.  Soy burger?

Filtering Live Messages
As mentioned earlier there can be large numbers of messages on the RS485 bus. Message Manager will allow you to filter live messages by their broad category. To do this click the hamburger menu at the upper right hand corner of the display. Then navigate to the logging tab.

On this tab we are interested in the Packets section. If no options in this section are enabled then no messages will be displayed in live view. As changes are made on this display, new incoming messages will be filtered in the message list. If an option has additional filtering options you can click on the filter icon located on that option button.
